### DISCUSSION TITLE
BlueRetro Advance config not saving with 2 devices

### DISCUSSION THREAD

**Discussion by shadowfire-36** [darthcloud/BlueRetro#1035] [General]
I have a bluetooth core hw 1 flashed with universal.bin 24.04(aliexpress), using a retro bit sega saturn controller that i setup with the core .
I have a sega saturn ,genesis, snes , and a dreamcast , I have been able to program the controllers to the core and use on each console one by one  .
what i have been trying to do is   set the defaults mappings for each change console and just move  the core to another console but i cant seem to get it to work,the core picks only the d-pad if i unplug the core to another comsole ,and  all of the button mappings i changed for each console do not seem to save . 
So i will start with the saturn and set its control , it works , If i set the saturn and dreamcast, the saturn works ,but if i plug to the dreamcast the buttons are not the ones i set in the advance config , they are still set for the saturn  
Am i missing a step in in setting this up having to controls for 4 consoles or is this not possible using the firmware i have flashed to the core ?

**Comment by Zabludilsya** [comment]
У меня вообще не сохраняет настройки для второго контроллера.Модуль не может сохранить настройки для подключения на два контроллера.

**Comment by darthcloud** [comment]
There is only one mapping on BlueRetro, it's not unique per console. so if you change it for  a console that affect all the console.
